larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -30,13 +30,7 @@ enum class KernelType {
   // kGenericRef: the reference implementation without im2col
   kGenericRef,
 
-  // kGenericOptimized: the impl. path using reference BGEMM kernels in RUY.
-  // TODO(arashb): the generic optimized needs to be redirected to the
-  // reference impl. of RUY kernels.
-  kGenericOptimized,
```

Comment:
This kernel type isn't implemented; we can add this back if we ever need to in the future.